src/components/Trilemma/index.tsx

```diff
@@ -81,7 +81,7 @@ const Trilemma = () => {
               justifyContent="flex-start"
               my={8}
             />
-            <DrawerCloseButton top={6} insetInlineEnd={6} />
+            <DrawerCloseButton top={6} right="unset" insetInlineEnd="6" />
```

Comment:
`right="unset"` does nothing here

```suggestion
            <DrawerCloseButton top={6} insetInlineEnd="6" />
```

Reply:
Yep, good catch.. this was added before we fixed the `inset*` issue which seems to have made these close buttons work more as-expected in RTL